Automattic/wp-calypso
# Stats: Remove Sticky Tab Logic.

This is a follow-up task to #10567.  Now that the default stats view is set to 'Day' - the `stats-sticky` logic is no longer used or needed.  This branch removes the old flux store, component, and usage of both.

__To Test__
- Open up stats, navigate between Insights, and day periods.  Ensure all things work as normal, no visual changes expected.
